website/docs/routes/game.md

```diff
@@ -1001,12 +1001,17 @@ Example：`https://www.iyingdi.com/tz/people/55547` ，id 是 `55547`
 
 <Route author="wy916787036" example="/vgn" path="/vgn/:platform?" paramsDesc={['平台，见下表，默认为全部']}>
 
-| Switch | PS4 | PC | Xbox |
-| ------ | --- | -- | ---- |
-| 1      | 2   | 3  | 4    |
+| Switch | PS4 | PC  | Xbox |
+| ------ | --- | --- | ---- |
+| 1      | 2   | 3   | 4    |
 
 </Route>
 
+## 游戏基因 {#you-xi-ji-yin}
+
+### 资讯 {#you-xi-ji-yin-zi-xun}
+
+<Route author="lone1y" example="/gamegene/news" path="/gamegene/news"/>
```

Comment:
docs cannot be built.
https://github.com/DIYgod/RSSHub/actions/runs/6180542576/job/16777150759

Guide: https://docs.rsshub.app/joinus/new-rss/add-docs

Reply:
resolve